downloader_app/tiff_downloader.py

```diff
@@ -62,26 +68,30 @@
 
 
 def download_tiffs(source, dates, point1, point2, opt=False):
-    """ 
-    Function responsible for the main calls. 
-    
+    """
+    Function responsible for the main calls.
     Inputs
     ------
     source: string
-        It is the name of the source from which we will take the captured images. 
+        It is the name of the source from which we will take the captured images.
         For example 'LandDAAC-v5-day' is a valid name. Other possible names may
         be seen with the command about('sources').
     dates: collection of timestamps
         The days of interest to download data.
     point1, point2: tuple
```

Comment:
for tuple, list, etc you can be more specific ... for example tuple(string), list[str, int] etc. you can use the same approach used by mypy